xml/ha_configuring_resources.xml

```diff
@@ -920,19 +920,39 @@
     secondary). Promotable clones must contain
     exactly one group or one regular resource.
    </para>
+   <para>
+    &pace; determines which instance of the clone to promote using one of the following
+    methods (or a combination of both):
+   </para>
+   <itemizedlist>
+     <listitem>
+       <para>
+         If the resource agent supports promotable clones, it automatically sets
+         <emphasis>promotion scores</emphasis>.
```

Comment:
How about something like: "If a resource agent supports promotable clones, it may have its own preference which instance to promote and set promotion scores accordingly by itself"?